rio_tiler/utils.py

```diff
@@ -218,6 +221,21 @@ def _get_descr(ix):
             masked=True,
         )
 
+        if expr is not None:
+            bands_names = tuple(set(re.findall(r"(?P<bands>b[0-9A]{1,2})", expr)))
```

Comment:
we can remove the `A` for this because the expression should only correspond to band indexes. Also maybe the limitation to 2 number might not be necessary

Reply:
Something like `(?P<bands>b[0-9]+)`?